Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "IntervalArithmetic"
 uuid = "d1acc4aa-44c8-5952-acd4-ba5d80a2a253"
 repo = "https://github.com/JuliaIntervals/IntervalArithmetic.jl.git"
-version = "0.17.2"
+version = "0.17.0"
```

Comment:
This isn't right. We should be increasing the version as you had before.

Reply:
Each minor change like this should increment the "patch version", i.e. the 3rd number.

Reply:
In master, it is 0.17.1 so I initially changed it to 0.17.2 but the tests were conflicting so I reverted the commit.
Should I keep it at 0.17.2?

Reply:
Yes keep at 0.17.2.
What do you mean by "the tests were conflicting"?